package.json

```diff
@@ -25,15 +26,16 @@
   },
   "homepage": "https://github.com/jonkemp/mediaquery-text",
   "dependencies": {
-    "cssom": "0.3.2"
+    "cssom": "0.4.1",
```

Comment:
We are facing the below error since this change was introduced. Could you please look into it.

14:39:54       [exec] > mediaquery-text@1.0.8 postinstall /var/lib/jenkins/workspace/...../nodejs/frontend/node_modules/mediaquery-text
14:39:54       [exec] > patch-package
14:39:54       [exec] 
14:39:54       [exec] Error: Patch file found for package cssom which is not present at node_modules/cssom

Reply:
Ok, thanks for letting me know.

Reply:
FYI, we use gulp-inline-css: ~3.3.1 module in our codebase. I see that you are also the contributor for that module.